test/unit/core/editor.js

```diff
@@ -897,6 +897,11 @@ describe('Editor', function () {
       expect(editor.getHTML(0, 11)).toEqual('&lt;b&gt;Test&lt;/b&gt;');
     });
 
+    it('multiline html', function () {
```

Comment:
What do we test here? "last line break should not be parsed on editor initialization"?
Please, make test name more strict - i didn't catch which scenario is tested here